portfolio/src/main/webapp/images.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Kathleen's Photo Gallery</title>
+    <link rel="stylesheet" href="style.css">
+    <style>
+        h1 {text-align: center; font-family: Georgia, serif;}
+        .center {text-align: center;}
+        div.gallery {border: 1px solid #ccc;}
```

Comment:
Can be best to put this css in the style.css instead of here.

Reply:
Yes, really great idea! It makes the code structure look clear.